spacy/pipeline/ner.pyx

```diff
@@ -40,6 +40,7 @@ DEFAULT_NER_MODEL = Config().from_str(default_model_config)["model"]
         "moves": None,
         "update_with_oracle_cut_size": 100,
         "model": DEFAULT_NER_MODEL,
+        "negative_samples_key": None
```

Comment:
I'm not sure it's a good idea to specify this parameter here at the component factory. Imagine sourcing an `ner` component from a base model - you won't be able to change this parameter in the config. It feels like it should be more of a reader thing?